fastapi/routing.py

```diff
@@ -249,6 +249,10 @@ def __init__(
             self.dependencies = []
         self.summary = summary
         self.description = description or inspect.cleandoc(self.endpoint.__doc__ or "")
+        # if a form feed (page break) is found in the description text, truncate
+        # description text to the content preceding the first form feed
+        if self.description and "\f" in self.description:
```

Comment:
No need to check for `"\f" in self.description`; the first result of `split` will just be the whole string in that case